test_apps/advection/README.md

```diff
@@ -0,0 +1,59 @@
+# VAPOR Particle Advection Benchmarking
+
+This directory has the benchmarking program for VAPOR's particle advection module
+
+## Build project
+
+The project has 1 external dependency apart from VAPOR
+-- The C++ Boost library [Program Options, File System]
+
+In order to find all the required VAPOR components the following variables must
+be provided to CMake
+
+```
+VAPOR_PREFIX : path to the VAPOR build/installation
+VAPOR_INCLUDE_DIR : path to the include directory in the VAPOR repository
+VAPORDEPS_PREFIX : path to where the VAPOR dependencies are located
+```
+A sample CMake command to build this project would look like
+
+```
+cmake -DVAPOR_PREFIX=/Users/abhishek/repositories/VAPOR/build \
```

Comment:
These macros point to files in your home directory. Is there anything special about these? I.e. have they been modified from the original?

Reply:
There's nothing special about the VAPOR build as such.
But I did not want to add building of this application with the test_apps, I've used a couple of Boost modules which are not available with VAPOR CMake stuff after being built.